src/plone/rest/tests/test_traversal.py

```diff
@@ -126,3 +126,7 @@ def test_json_request_to_view_namespace_returns_view(self):
         self.portal[self.portal.invokeFactory('Folder', id='folder1')]
         obj = self.traverse('/plone/folder1/@@folder_contents')
         self.assertTrue(IBrowserView.providedBy(obj), 'IBrowserView expected')
+
+    def test_json_request_to_portal_resource_returns_page_template(self):
+        obj = self.traverse('/plone/portal_resources')
```

Comment:
@displacedaussie this is not a POST request. Are you sure this test fails without the fix?